[misc] Add new issues templates (#3390)
taichi-dev/taichi: .github/ISSUE_TEMPLATE/document.md

---
name: Doc
about: Have doubts or suggestions about Taichi Docs
title: ''
labels: doc
assignees: ''

---

<!--

> Please describe any errors, ambiguities, and any areas that you think can be improved in the documentation.

Reviewer: ```suggestion
Please describe any errors, ambiguities, or other improvement opportunities that you can find in the documentation.
```
